.github/workflows/build-system-test-react-native.yml

```diff
@@ -117,42 +119,18 @@ jobs:
           $ANDROID_HOME/platform-tools/adb shell settings put global animator_duration_scale 0.0
 
       - name: Create MegaApp ${{ env.MEGA_APP_NAME }}
-        run: ./.github/scripts/mega-app-create-app.sh
+        run: ./.github/scripts/mega-app-create-app.sh --build-tool ${{ matrix.build-tool }} --build-tool-version ${{ matrix.build-tool-version }} --language ${{ matrix.language }} --name ${{ env.MEGA_APP_NAME }} --framework ${{ matrix.framework }} --framework-version ${{ matrix.framework-version }}
```

Comment:
I see that this is getting pretty long (which is coming from my own suggestion 😅 ). 

Doesn't have to be this PR, but I think creating `alias`es in `package.json` to ease this a little bit and for future developers:

```json
"angular:latest": "./github/scripts/mega-app-create-app.sh --framework angular --framework-version 14",
"angular:14": "./github/scripts/mega-app-create-app.sh --framework angular -framework-version 14",
```

Reply:
Good idea! I was thinking of adding a GH env, but I think your suggestion works better because it may solve some of my headache on creating the mega-apps on local

Reply:
Discussed offline, we can scope this to the second PR.